initial/application/build.gradle

```diff
@@ -1,30 +1,19 @@
-buildscript {
-    ext { springBootVersion = '2.1.6.RELEASE' }
-    repositories { mavenCentral() }
-    dependencies { classpath("org.springframework.boot:spring-boot-gradle-plugin:${springBootVersion}") }
-}
-
 plugins {
-    id "io.spring.dependency-management" version "1.0.5.RELEASE"
+	id 'org.springframework.boot' version '2.1.8.RELEASE'
+	id 'io.spring.dependency-management' version '1.0.8.RELEASE'
+	id 'java'
 }
 
-apply plugin: 'java'
-apply plugin: 'eclipse'
-apply plugin: 'org.springframework.boot'
-apply plugin: 'io.spring.dependency-management'
+group = 'com.example'
+version = '0.0.1-SNAPSHOT'
+sourceCompatibility = '1.8'
 
-bootJar {
-    baseName = 'gs-multi-module-application'
-    version = '0.0.1-SNAPSHOT'
+repositories {
+	mavenCentral()
 }
-sourceCompatibility = 1.8
-
-repositories { mavenCentral() }
 
 dependencies {
-    compile('org.springframework.boot:spring-boot-starter-actuator')
-    compile('org.springframework.boot:spring-boot-starter-web')
-    compile project(':library')
-    testCompile('org.springframework.boot:spring-boot-starter-test')
+	implementation 'org.springframework.boot:spring-boot-starter-actuator'
+	implementation 'org.springframework.boot:spring-boot-starter-web'
+	testImplementation 'org.springframework.boot:spring-boot-starter-test'
```

Comment:
Missing dependency on library? I guess that's OK for the initial?

Reply:
(I thought I answered this, but I don't see it. Sorry if there are two answers.)

I thought I would use only what the Initializr generated in the initial project and have the complete project contain the bits as they look after editing.